packages/playground/src/components/project-settings/members-list.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useReducer } from 'react'
+import {
+  Icon,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+  Text,
+  Avatar,
+  AvatarImage,
+  AvatarFallback,
+  Badge,
+  Button,
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuGroup,
+  DropdownMenuItem,
+  DropdownMenuSeparator,
+  DropdownMenuShortcut,
+  DropdownMenuTrigger
+} from '@harnessio/canary'
+import { getInitials } from '../../utils/utils'
+import { FormEditMemberDialog } from './form-member-edit-dialog'
+import { FormDeleteMemberDialog } from './form-member-delete-dialog'
+import { dialogStateReducer, initialDialogState } from './members-reducers/dialog-state-reducers'
+
+interface MembersProps {
+  display_name: string
+  role: string
+  email: string
+  timestamp?: string
+  avatarUrl?: string
+}
+
+interface PageProps {
+  members: MembersProps[]
+}
+
+export const MembersList = ({ members }: PageProps) => {
+  const [dialogState, dispatch] = useReducer(dialogStateReducer, initialDialogState)
```

Comment:
Looks like this is only for `playground` to simulate user deletion, update etc. This will not be there when it's integrated in `gitness`.